bedrock/newsletter/templates/newsletter/mozilla-and-you.html

```diff
@@ -18,6 +18,12 @@ <h1 class="large">{{ _('Read all about it in our <span>newsletter</span>') }}</h
 {% endblock %}
 
 {% block success_confirm %}
-  <p>{% trans %}Thanks! Please check your inbox to confirm your subscription.{% endtrans %}</p>
-  <p>{% trans %}You'll receive an email from mozilla@e.mozilla.org to confirm your subscription. If you don't see it, check your spam filter. You must confirm your subscription to receive our newsletter.{% endtrans %}</p>
+  <h3>{{ _('Thanks! ') }}</h3>
+  <p>
+  {% trans %}
+    If you haven’t previously confirmed a subscription to a Mozilla-related
+    newsletter you may have to do so. Please check your inbox or your spam filter
+    for an email from us.
+  {% endtrans %}
+  </p>
```

Comment:
Since this template extends `one_newsletter_signup.html` (which has the updated copy), instead of repeating the thank you message, you can import the parent template's content for this block alongside additions. Something like:

```
{% block success_confirm %}
  <h3>{{ _('Thanks!') }}</h3>
  {{ block.super }}
{% endblock %}
```

Test that out, but I believe it'll work.